src/main/java/com/cupid/jikting/like/controller/LikeController.java

```diff
@@ -19,13 +20,13 @@ public class LikeController {
     private final JwtService jwtService;
 
     @GetMapping("/received")
-    public ResponseEntity<List<LikeResponse>> getAllReceivedLike(@RequestHeader("Authorization") String token) {
-        return ResponseEntity.ok().body(likeService.getAllReceivedLike(jwtService.extractValidMemberProfileId(token)));
+    public ResponseEntity<List<LikeResponse>> getAllReceivedLike(@AuthVariable Long memberProfileId) {
+        return ResponseEntity.ok().body(likeService.getAllReceivedLike(memberProfileId));
     }
 
     @GetMapping("/sent")
-    public ResponseEntity<List<LikeResponse>> getAllSentLike(@RequestHeader("Authorization") String token) {
-        return ResponseEntity.ok().body(likeService.getAllSentLike(jwtService.extractValidMemberProfileId(token)));
+    public ResponseEntity<List<LikeResponse>> getAllSentLike(@AuthVariable Long memberProfileId) {
```

Comment:
p3. 어노테이션을 축약하지 않고, 인증된 값이라는 걸 명시해주는건 어떨까요?

```suggestion
    public ResponseEntity<List<LikeResponse>> getAllSentLike(@AuthorizedVariable Long memberProfileId) {
```